Dockerfile.daemon

```diff
@@ -1,6 +1,6 @@
-FROM node:16
+FROM node:16 as ceramic
 
-RUN apt-get update && apt-get install -y netcat
+RUN apt-get update && apt-get install -y netcat gettext-base
```

Comment:
What do we use gettext for?

Reply:
This installs the `envsubt` command which is used in the container to template a config file.